src/HelperLocator.php

```diff
@@ -3,11 +3,11 @@
 
 class HelperLocator
 {
-    static public function new(Escape $escape = null) : self
+    static public function new(Escape $escape = null, array $factories = []) : static
```

Comment:
Hey @koriym ,
I am wondering what is the change from self to static to do. Also this one is returning self. So the return type should also be self right ?